components/dashboard/src/components/SelectIDEComponent.tsx

```diff
@@ -148,7 +148,7 @@ function IdeOptionElementInDropDown(p: IdeOptionElementProps): JSX.Element {
     return (
         <div className="flex" title={option.title}>
             <div className="mx-2 my-2">
-                <img className="w-8 filter-grayscale self-center" src={option.logo} alt="logo" />
+                <img className="w-8 h-8 filter-grayscale self-center" src={option.logo} alt="logo" />
```

Comment:
Snuck this change in here, as I noticed a layout shift when opening the ide selection dropdown that can be avoid w/ a height set on the editor icon images.